rollup.utils.js

```diff
@@ -84,6 +84,11 @@ function babelPluginReplaceVersionPlaceholder() {
               return;
             }
 
+            // Skip for experimental releases
+            if (version.startsWith("0.0.0")) {
+              return;
```

Comment:
This would get us into a loop since we'd be replacing the placeholder value of `0` with a new value of `0`